content/includes/ngf/installation/manifests/api-resources.md

```diff
@@ -0,0 +1,22 @@
+---
+---
+
+{{< call-out "note" >}} If you have already installed Gateway API resources in your cluster, ensure they are a version [supported by NGINX Gateway Fabric]({{< ref "/ngf/overview/technical-specifications.md" >}}) {{< /call-out >}}
+
+To install the Gateway API resources, use `kubectl kustomize`:
+
+```shell
+kubectl kustomize "https://github.com/nginx/nginx-gateway-fabric/config/crd/gateway-api/standard?ref=v{{< version-ngf >}}" | kubectl apply -f -
+```
+
+{{< details summary="Example output" >}}
+
+```text
+customresourcedefinition.apiextensions.k8s.io/gatewayclasses.gateway.networking.k8s.io created
+customresourcedefinition.apiextensions.k8s.io/gateways.gateway.networking.k8s.io created
+customresourcedefinition.apiextensions.k8s.io/grpcroutes.gateway.networking.k8s.io created
+customresourcedefinition.apiextensions.k8s.io/httproutes.gateway.networking.k8s.io created
+customresourcedefinition.apiextensions.k8s.io/referencegrants.gateway.networking.k8s.io created
```

Comment:
This is going to continually change as Gateway API adds new CRDs. Doesn't seem maintainable.

Reply:
Unless the new CRDs are being added daily, I don't know why there wouldn't be a problem updating the example output periodically.

They're meant to give an impression of output, and are hidden by default.